lib/collection.js

```diff
@@ -381,7 +389,7 @@ Collection.prototype.find = deprecateOptions(
         promiseLibrary: this.s.promiseLibrary,
         ignoreUndefined: this.s.options.ignoreUndefined
       },
-      { validationLevel: DEFAULT_VALIDATION }
+      { optionsValidationLevel: this.s.optionsValidationLevel }
```

Comment:
now that you have that very slick inherited property above, you need to actually _use_ it here :) this should be `{ optionsValidationLevel: this.optionsValidationLevel }`, for all cases below